### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by MicroverseNelson**
## DESKTOP VERSION
- added media queries for (min-width: 768)
- added media query on all sections : hero, works, aboutme, contact and footer section. 
- made changes as per the review:
                         - text adjustment 
                         - hero image adjustment
                         - hover over effects
                         - projects display upon hoover over
                         - link project button to git repo

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -205,9 +220,12 @@ <h2 class="contact-form-title">Get In Touch</h2>
...
-        <input type="text" placeholder="Email address">
-        <textarea name="message" id="messageID" cols="30" rows="10" placeholder="Enter text here"></textarea>
+        <div class="form-input-names">
+          <input name="name" type="text" placeholder="First name" maxlength="30" required>
+          <input name="name" type="text" placeholder="last name" maxlength="30" required>
+        </div>
+        <input name="email" type="email" placeholder="Email address" maxlength="30" required>
+        <textarea name="message" id="messageID" cols="30" rows="10" placeholder="Enter text here" maxlength="500" required></textarea>
         <button type="submit" class="getintouchbtn">Get in touch</button>
       </div>
```
Comment (zieeco) [inline index.html:230]: - [ ] Hey, kindly adjust the styling in your form to match the Figma design..
 **YOUR DESIGN**

clear




**FIGMA DESIGN**


FORMS

File: index.html [hunk 2]
```diff
@@ -47,115 +57,120 @@ <h3>I’am a Software Developer</h3>
...
+            Hello I’m a software developer! I can help you build a product , feature
+            or website Look through some of my work and experience! If you like what
+            you see and have a project you need coded, don’t hestiate to contact me.
+        </p>
+        <button class="aboutme-resumebtn" type="submit">Get My Resume</button>
+        <img class="divider" src="./images/Devider.png" alt="" />
+      </div>
       <div class="aboutme-technologies">
         <ul class="technologies-category">
           <li class="first">
```
Comment (zieeco) [inline index.html:176]: - [ ] Hey, great job with your design, however, your design is not yet close to the Figma design. Please consider this little adjustment to make it even better..

**YOUR DESIGN**

now


**FIGMA DESIGN**

noee

**WHAT TO IMPROVE**
- Image should be aligned to the left
- Increase the space from top and bottom for the horizontal line
- The `Languages, Frameworks and Skills ` are aligned in the center.
- The about section is placed in the center, which aligns with the works section
- The paragraph in the about section takes up 60% width of the parent while the image part takes up the remaining 40% percent

**Please kindly make this adjustment to make it look better** :pray:

File: main.css [hunk 3]
```diff
@@ -344,3 +346,356 @@
...
+    margin: 50px 50px 50px 50px;
+    gap: 14px;
+  }
+
+  .projects-section .project-item {
+    
+    height:300px;
+    padding: 50px 10px 10px 10px;
+    color:#fff;
+  }
```
Comment (zieeco) [inline main.css:502]: - [ ] Hey, your project card is missing the hover effect. When the project is hovered on with the mouse the description and dark opacity should disappear and the `See Project button` appears at the bottom of the image which should take the user to the Live Demo of your portfolio or to the GitHub repository. 
Please, kindly implement this feature.

File: main.css [hunk 4]
```diff
@@ -344,3 +346,356 @@
...
+    color: #091E42;
+  } 
+
+  .works-section .works-text {
+    display: flex;
+    flex-direction: row;
+    gap: 14px;
+    margin: 0 50px 50px 50px;
+  }
+
```
Comment (zieeco) [inline main.css:465]: - [ ] Hey, you're doing great with your design, however, you need to add this adjustment.
The image with the `.placholder` class is big and takes up 60% of the width in your `.works-text` container and the rest content takes up the remaining. 
Right now, your content is taking the width of the image. Please adjust your styling to fix this.

File: main.css [hunk 5]
```diff
@@ -344,3 +346,356 @@
...
+  .hero-section .hero-social-links {
+    margin-left:16px;
+    list-style: none;
+    padding-top: 100px;
+  }
+
+  .hero-section .hero-social-links img {
+    width: 60%;
+    margin-top: 10px;
+  }
```
Comment (zieeco) [inline main.css:440]: - [ ] Hey, please kindly adjust the position of your social links by moving it up to the top a little so it aligns in the middle of your welcome section.

**YOUR DESIGN**

contact


**FIGMA DESIGN**

TRUE

- Kindly make your image align with the end of the section and increase its width.

**YOUR DESIGN**

IMAGE

**FIGMA DESIGN**

HMM

File: index.html [hunk 6]
```diff
@@ -205,9 +220,12 @@ <h2 class="contact-form-title">Get In Touch</h2>
...
-        <input type="text" placeholder="Full name">
-        <input type="text" placeholder="Email address">
-        <textarea name="message" id="messageID" cols="30" rows="10" placeholder="Enter text here"></textarea>
+        <div class="form-input-names">
+          <input name="name" type="text" placeholder="First name" maxlength="30" required>
+          <input name="name" type="text" placeholder="last name" maxlength="30" required>
+        </div>
+        <input name="email" type="email" placeholder="Email address" maxlength="30" required>
+        <textarea name="message" id="messageID" cols="30" rows="10" placeholder="Enter text here" maxlength="500" required></textarea>
         <button type="submit" class="getintouchbtn">Get in touch</button>
```
Comment (zieeco) [inline index.html:229]: - [ ] Please, kindly add the `hover, active, and disabled` CSS pseudo-classes to all your buttons.

button

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @MicroverseNelson 


Great work on your design 👏 
You've done well implementing this design, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by m05e5 — APPROVED** [review]
# Hi @MicroverseNelson , 

### Status: Approved ✔️
goodjob


:+1: You have done all the required changes asked by the previous reviewer. Well Done.

Your project is complete! 
Congratulations! 🎉 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- _(optional)_ You really did a great job. I recommend fitting the design as much as possible
**your design**
image
**Figma design**
image


**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
-----

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_m05e5_**

------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**